neo4j-store/src/main/kotlin/org/ostelco/prime/storage/graph/Neo4jStore.kt

```diff
@@ -898,12 +916,74 @@ object Neo4jStoreSingleton : GraphStore {
                             ?: NotFoundError(type = simProfileEntity.name, id = iccId).left().bind<SimProfile>()
 
                     update { simProfile.copy(alias = alias) }.bind()
-                    AuditLog.info(customerId = customerId, message = "Updated alias of SIM Profile")
+                    AuditLog.info(customerId = customerId, message = "Updated alias of SIM Profile (iccId = $iccId)")
+                    org.ostelco.prime.model.SimProfile(
```

Comment:
Here a sim profile is being mapped to an almost identical sim profile object.  There should be a comment stating clear what kind of mapping this is and why it is necessary.  Also since most of the fields are being copied, it would be clearer if a builder with defaults from the original instance was being used. That would make it very obvious exactly  what fields are being changed.  In this case I believe it is "status" and "eSimActivationCode", and in the code  starting at line 945 it is nothing?

Reply:
Comments are added at top-level instead of every line where it is used.  Also, used different alias name for the class in import since we have 2 set of classes with the same names.